### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by reohpsy**
I tweak the icons with href linking

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -36,10 +36,10 @@ <h1 class="text-4xl font-bold text-gray-800">
                             That could help my environment and the world at large
                         </p> 
                         <div class="inline-flex space-x-4 text-xl" >
-                            <ion-icon name="logo-twitter" class="text-4xl text-purple-500"></ion-icon>
-                            <ion-icon name="mail-unread-outline" class="text-4xl text-purple-500"></ion-icon>
-                            <ion-icon name="logo-slack" class="text-4xl text-purple-500"></ion-icon>
-                            <ion-icon name="logo-github" class="text-4xl text-purple-500"></ion-icon>
+                           <a href="https://twitter.com/reohpsy"><ion-icon name="logo-twitter" class="text-4xl text-purple-500"></ion-icon></a>
```
Comment (reohpsy) [inline index.html:39]: This will redirect to my twitter profile, insert this instead for the href link
<a href="https://twitter.com/_ellahhh__">

**Comment by oghene-ella** [comment]
lmao, okay.. I will work on that, thank you!

File: index.html (hunk 1 above)
Comment (reohpsy) [inline index.html:39]: 1. https://twitter.com/_ellahhh__
use this for your twitter href link

File: index.html [hunk 2]
```diff
@@ -36,10 +36,10 @@ <h1 class="text-4xl font-bold text-gray-800">
...
+                          <a href="https://rowlandopeyemi@gmail.com"><ion-icon name="mail-unread-outline" class="text-4xl text-purple-500"></ion-icon></a>
```
Comment (reohpsy) [inline index.html:40]: and this will be your mail 
i.e oghenekaro57@gmail.com with href linking

File: index.html [hunk 3]
```diff
@@ -36,10 +36,10 @@ <h1 class="text-4xl font-bold text-gray-800">
...
+                      <a href="https://slack.com/reohpsy"><ion-icon name="logo-slack" class="text-4xl text-purple-500"></ion-icon></a>
```
Comment (reohpsy) [inline index.html:41]: slack link can be left blank unless you have a workspace link.

just put <a href="#">.............

File: index.html [hunk 4]
```diff
@@ -36,10 +36,10 @@ <h1 class="text-4xl font-bold text-gray-800">
...
+                       <a href="https://github.com/reohpsy"><ion-icon name="logo-github" class="text-4xl text-purple-500"></ion-icon></a>
```
Comment (reohpsy) [inline index.html:42]: lastly this shld redirect to your account too

https://github.com/oghene-ella inside the href tag

**Review by reohpsy — COMMENTED** [review]
https://oghene-ella.github.io/My_Portfolio/

hover the icons on your page to see changes

**Pull request merged**